homeassistant/components/meteo_france/manifest.json

```diff
@@ -1,6 +1,7 @@
 {
   "domain": "meteo_france",
   "name": "Météo-France",
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/meteo_france",
   "requirements": ["meteofrance==0.3.7", "vigilancemeteo==3.0.0"],
   "dependencies": [],
```

Comment:
Maybe add yourself as codeowner?

Reply:
Since they don't seems active for now, I will.